hop/io.py

```diff
@@ -218,7 +218,7 @@ def from_format(data, format, deserialize=True):
                 return from_format(old["content"], old["format"], deserialize=False)
             # not labeled according to our scheme, but it is valid JSON
             return models.JSONBlob(content=old)
-        except json.JSONDecodeError:  # if we can't tell what the data is, pass it on unchanged
+        except (UnicodeDecodeError, json.JSONDecodeError):  # if we can't tell what the data is, pass it on unchanged
```

Comment:
The linter becomes unhappy about the length of this line; the comment can just be moved down onto a line of its own to fix that.